src/routes/concepts/refs.mdx

```diff
@@ -16,40 +18,59 @@ As elements with the same selectors are added and removed from the DOM, the firs
 
 ## JSX as a value
 
-When wanting to directly access the DOM element, you can use JSX as a value and assign it to a variable.
+You can also use JSX as a value and assign it to a variable.
 
-```jsx
+```tsx
 function Component() {
-	const myElement = <div>My Element</div>
+	const myElement = <p>My Element</p>
 
 	return <div>{myElement}</div>
 }
 ```
 
-This can be a convenient way to access the DOM element.
-It lets you access the element directly without having to wait until it is attached to the DOM, and it can be used multiple times without having to worry about duplicate selectors.
+This allows you to create and access DOM elements similar to using [`document.createElement`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) 
+without having to wait until it is attached to the DOM. It can be used multiple times without having to worry about duplicate selectors.
 
-The downside to this approach is it separates the element and any child elements that are created from it from the rest of the DOM.
-This removes the ability to access the element directly from the variable, and reading through components that use JSX as a value can be more difficult to understand.
+The downside to this approach is it separates the element and any child elements from the rest of the 
+JSX structure, and it can make the code harder to read and understand.
```

Comment:
```suggestion
The downside to this approach is it separates the element and any child elements from the rest of the JSX structure.
This makes reading through the components more difficult to read and understand. 
```

Reply:
@LadyBluenotes this change introduces a redundant use of "reading" and "read" which doesn't sound quite right.

Maybe this instead?

```suggestion
The downside to this approach is it separates the element and any child elements from the rest of the 
JSX structure. 
This makes the component's JSX structure more difficult to read and understand.
```

Reply:
Yes that works.